nuxt.config.js

```diff
@@ -204,6 +204,7 @@ export default {
     dsn:
       process.env.SENTRY_DSN ||
       'https://3f3e05dbe6994c318d1bf1c8bfcf71a1@o288582.ingest.sentry.io/1525413',
+    disabled: process.env.NODE_ENV === 'development',
```

Comment:
```suggestion
    disabled: process.env.NODE_ENV === 'development',
    environment: process.env.NODE_ENV
```

Reply:
@obulat this is how we'd add the environment support, which would be either `'staging'|'production'`.

Reply:
And a note, this is happening by default but it's probably better for us to be explicit about it.